lua/vex_library/modules/server/e2helperfuncs.lua

```diff
@@ -16,19 +16,89 @@ vex.registerExtension = function(name,enabled,helpstr,...)
     E2Lib.RegisterExtension(name,enabled,helpstr,...)
 end
 
+vex.registerConstant = function(name, value, ...)
```

Comment:
- [x] Added the `vex.registerConstant`. A small utility to help with syntax while registering E2 constant at the same time.